content/blog/set-up-apache-skywalking-for-k8s-monitoring-in-hpe-greenlake-for-private-cloud-enterprise.md

```diff
@@ -0,0 +1,368 @@
+                separator: ;
+                regex: (.+)
+                target_label: node
+                replacement: $$1
+                action: replace
+          # @feature: kubernetes-monitor; configuration to scrape Kubernetes Endpoints metrics
+          - job_name: kube-state-metrics
+            metrics_path: /metrics
+            kubernetes_sd_configs:
+            - role: endpoints
+            relabel_configs:
+            - source_labels: [__meta_kubernetes_service_label_app_kubernetes_io_name]
+              regex: kube-state-metrics
+              replacement: $$1
+              action: keep
+            - action: labelmap
+              regex: __meta_kubernetes_service_label_(.+)
+            - source_labels: []
+              target_label: cluster
+              replacement: cfe-iac-clu
+```
+
+##### 2. Create a YAML file *otel-collector-deploy.yaml* for the *OpenTelemetry* collector deployment:
+
+```markdown
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: otel-deployment-kubernetes-monitor
+  labels:
+    app: otel-kubernetes-monitor
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: otel-kubernetes-monitor
+  template:
+    metadata:
+      labels:
+        app: otel-kubernetes-monitor
+      annotations:
+        sidecar.istio.io/inject: "false"
+    spec:
+      serviceAccountName: otel-sa-kubernetes-monitor
+      containers:
+        - name: otel-kubernetes-monitor
+          image: otel/opentelemetry-collector:0.50.0
+          command:
+            - "/otelcol"
+            - "--config=/conf/otel-collector-config.yaml"
+          volumeMounts:
+            - name: otel-collector-config-vol-kubernetes-monitor
+              mountPath: /conf
+      volumes:
+        - name: otel-collector-config-vol-kubernetes-monitor
+          configMap:
+            name: otel-collector-conf-kubernetes-monitor
+            items:
+              - key: otel-collector-config
+                path: otel-collector-config.yaml
+```
+
+##### 3. Deploy the OpenTelemetry collector:
+
+```markdown
+$ kubectl apply -f otel-collector-config.yaml -n skywalking
+$ kubectl apply -f otel-collector-deploy.yaml -n skywalking
+```
+
+```markdown
+$ kubectl  get all -n skywalking -l app=otel-kubernetes-monitor
+NAME                                                      READY   STATUS    RESTARTS   AGE
+pod/otel-deployment-kubernetes-monitor-798cdd8486-gz885   1/1     Running   0          93d
+
+NAME                                                 READY   UP-TO-DATE   AVAILABLE   AGE
+deployment.apps/otel-deployment-kubernetes-monitor   1/1     1            1           96d
+
+NAME                                                            DESIRED   CURRENT   READY   AGE
+replicaset.apps/otel-deployment-kubernetes-monitor-798cdd8486   1         1         1       96d
+```
+
+After all setup steps are finished, the Kubernetes metrics will be showing up in SkyWalking UI, under *Kubernetes* tab:
```

Comment:
Rephrase the second part of this sentence to "... the Kubernetes metrics **will show up** in **the** SkyWalking UI, under **the** Kubernetes tab:"